aws/cloudformation/iam.yml.erb

```diff
@@ -835,6 +835,90 @@ roles.each do |name, config| %>
     Type: AWS::IAM::ServiceLinkedRole
     Properties: 
       AWSServiceName: logger.cloudfront.amazonaws.com
+
+  # ------------------------------------------------------------
+  # Create one AWS IAM User for each type of marketing sites environment that GitHub Actions will create, with
+  # permissions needed to create/update/delete a marketing sites CloudFormation Stack.
+  # ------------------------------------------------------------
+  <%
+    marketing_sites_environment_type_configuration = {
+      development: {
+        base_domain_name: 'dev-code.org'
+      },
+      test: {
+        base_domain_name: 'test-code.org'
+      },
+      production: {
+        base_domain_name: 'code.org'
+      }
+    }
+    marketing_sites_environment_type_configuration.each do |environment_type, configuration|
+  %>
+  # GitHub Actions authenticates as this user to deploy marketing sites Stacks.
+  GithubDeployerMarketingSites<%=environment_type%>:
+    Type: AWS::IAM::User
```

Comment:
With the new OIDC implementation, we don't need an IAM User, just the IAM Role (not sure if that is being done in a different PR)